flash_notebooks/finetuning/image_classification.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p> </p><p>Maybe</p><p>In this example we will be downloading the Hymenoptera dataset from a URL, and save that in a 'data' directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done !</p>